http-module/src/main/resources/dw/io/http/Client.dwl

```diff
@@ -23,7 +23,8 @@ var DEFAULT_HTTP_CLIENT_CONFIG = {
   decompress: true,
   tls: {
     insecure: false
-  }
+  },
+  useProxyProperties: false
```

Comment:
Should we expose this to the customers? Is there a way to turn it off in other clients? I think this is a bug and we must say that all the HttpClients should honor this. Adding this flag means that on clients that this was well done we need to find a way to turn it off... for example in mule in the future and in SF core